diff --git a/dart/external/imgui/CMakeLists.txt b/dart/external/imgui/CMakeLists.txt
--- a/dart/external/imgui/CMakeLists.txt
+++ b/dart/external/imgui/CMakeLists.txt
@@ -21,4 +21,7 @@
 if(CMAKE_COMPILER_IS_GNUCXX)
   target_compile_options(${target_name} PRIVATE -w)
 endif()
+if(CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+  target_compile_options(${target_name} PRIVATE -Wno-nontrivial-memcall)
+endif()
 target_compile_definitions(${target_name} PUBLIC IMGUI_DISABLE_OBSOLETE_FUNCTIONS)
